stack/database.py

```diff
@@ -117,8 +144,8 @@
     DBInstanceClass=Ref(db_class),
     DBInstanceIdentifier=Ref(AWS_STACK_NAME),
     Engine="postgres",
-    EngineVersion="9.4.5",
-    MultiAZ=True,
+    EngineVersion=Ref(db_engine_version),
+    MultiAZ=Ref(db_multi_az),
```

Comment:
@tobiasmcnulty Not sure this makes sense since the `RDS` instance is bound to two subnets in different `AZ` c.f.: https://github.com/Omixy/ecs/blob/master/stack/database.py#L108